### PR TITLE
Card

### PR DESCRIPTION
**Pull request by Lokeshkhati**
image

image

image

image

image

Kindly review this card component

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
cards looks great to me, if possible replace all px with rem in padding.

File: css/components/card.css [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  cursor: pointer;
+  font-size: 2rem;
+  padding: 8px;
+}
+.brand-info i:hover {
+  border-radius: 50%;
+  background-color: #f1f5f9;
+}
+
+#close-icon {
```
Comment (kalyandindu) [inline css/components/card.css]: I suggest to make use of classes rather than id's for lower specificity and reusability

**Review by kalyandindu — APPROVED** [review]
Suggestion: To achieve more consistency make use of few variables for spacing, fonts size etc. Also few flex utility classes would help more to the code.

File: css/components/card.css (hunk 1 above)
Comment (Lokeshkhati) [reply css/components/card.css]: Thanks for your suggestion

**Review by kalyandindu — APPROVED** [review]

**Pull request merged**